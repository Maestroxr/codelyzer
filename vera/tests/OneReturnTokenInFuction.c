
/*OK*/

int functionOne()
{
	return 1;
}

int functionTwo()
{
	return 2;
}

/*--------*/


/*PROBLEM*/
int functionThree()
{
	if (12>10)
	{
		return 3;
	}	
	return 3;
}

int functionFour()
{
	int x = 0;
	for (i = 0; i <10; i++) 
	{
		return x++;
	}
	if (x > 10)
	{
		return x-1;
	}
	else
	{
		return x+1;
	}
	return x;
}

/*--------*/



