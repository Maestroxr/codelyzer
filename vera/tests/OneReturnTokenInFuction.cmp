C:\Work\AutoAssessment\codelyzer\vera\tests\OneReturnTokenInFuction.c:24: Only one 'return' token should be used in a function. Found another in line:22
C:\Work\AutoAssessment\codelyzer\vera\tests\OneReturnTokenInFuction.c:36: Only one 'return' token should be used in a function. Found another in line:32
C:\Work\AutoAssessment\codelyzer\vera\tests\OneReturnTokenInFuction.c:40: Only one 'return' token should be used in a function. Found another in line:36
C:\Work\AutoAssessment\codelyzer\vera\tests\OneReturnTokenInFuction.c:42: Only one 'return' token should be used in a function. Found another in line:40
