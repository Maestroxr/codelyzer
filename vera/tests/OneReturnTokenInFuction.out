NEXT> value:10 line:20 column:8
NEXT> value:> line:20 column:7
NEXT> value:12 line:20 column:5
NEXT> value:10 line:34 column:9
NEXT> value:> line:34 column:7
NEXT> value:x line:34 column:5
C:\Work\AutoAssessment\codelyzer\vera\tests\OneReturnTokenInFuction.c:24: Only one 'return' token should be used in a function. Found another in line:22
C:\Work\AutoAssessment\codelyzer\vera\tests\OneReturnTokenInFuction.c:36: Only one 'return' token should be used in a function. Found another in line:32
C:\Work\AutoAssessment\codelyzer\vera\tests\OneReturnTokenInFuction.c:40: Only one 'return' token should be used in a function. Found another in line:36
C:\Work\AutoAssessment\codelyzer\vera\tests\OneReturnTokenInFuction.c:42: Only one 'return' token should be used in a function. Found another in line:40
